### DISCUSSION TITLE
Улучшить возможность выхода из приложения

### DISCUSSION THREAD

**Discussion by lelicopter** [Storik4pro/goodbyeDPI-UI#56] [Ideas]
В случае, если при нажатии на крестик выполняется сворачивание (а не закрытие), возникает сложность с выходом из приложения:
При развернутом окне нет иконки в трее - т.е. чтобы выйти, надо сначала закрыть (свернуть) окно, чтобы появилась икона. Потом на иконке выбрать "Выход". При режиме "сворачивания в трей" никак иначе не выйти из приложения.

Т.е. идея в том, чтобы или добавить пункт Выход в меню закладок, или никогда не убирать иконку из трея (даже при развернутом приложении), если включен признак всегда сворачивать в трей.

**Comment by Storik4pro** [comment]
Вы можете изменить поведение при нажатии на кнопку `"закрыть"` на выход из приложения и сворачивать приложение в трей через кнопку `"свернуть"`

**Comment by lelicopter** [comment]
Нет, свернуть надо в таскбар, а закрыть - в трей. Это обычно поведение "свернуть" и "закрыть" у большинства приложений (которые закрываются в трей). Например, Телеграмм, и другие.

**Comment by Storik4pro** [comment]
Было добавлено в [обновлении 1.2.6](https://github.com/Storik4pro/goodbyeDPI-UI/releases/tag/1.2.6)
